setup/src/Magento/Setup/Module/Di/Code/Reader/ClassesScanner.php

```diff
@@ -16,13 +17,32 @@ class ClassesScanner implements ClassesScannerInterface
     protected $excludePatterns = [];
 
     /**
+     * @var array
+     */
+
+    private $fileResults = [];
+
+    /**
+     * @var string
+     */
+
+    private $generationDirectory;
+
+    /**
      * @param array $excludePatterns
+     * @param string $generationDirectory
      */
-    public function __construct(array $excludePatterns = [])
+
```

Comment:
Please remove all such empty lines. I'm not sure it is enforced by static tests currently but is definitely inconsistent with Magento 2 codebase.